batched-tx-disperse-multisend-py/.github/README.md

```diff
@@ -0,0 +1,98 @@
+# Disperse / Multisend batched transactions
+
+## Description
+
+Disperse / Multisend are apps used to distribute ERC-20 to multiple addresses in one transaction.
+Although they are useful to reduce gas fees by sending transactions in batches, many use them as part of their scam schemes. 
+
+The goal of this bot is to alert when someone is using any of these apps in order to send native tokens or any other ERC-20 token.
+
+## Supported Chains
+
+The bot is specific to the contracts [`Disperse`][etherscan-contract-disperse] and [`Multisend`][etherscan-contract-multisend].
+
+So it runs on a single chain:
+
+- Ethereum
+
+## Alerts
+
+The bot only emits `info` alerts:
```

Comment:
could you also incorporate the anomaly score? pls take a look at other bots in the starter kit on how this is done.

Reply:
I'll look at it!